chrome_extension/background.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2019 The Project Oak Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+// TODO: Generate an ECDSA key pair, and use the private key for authentication and the public key
+// as Oak label.
+//
+// See https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/sign
+
+// Ids of tabs that have entered Oak mode.
+const enabled_tabs = new Set();
```

Comment:
nit: I will caution that camlCase is the most common naming convention in JavaScript land, but I'm not precious about it :)

Reply:
ahh, I personally dislike camelCase, but since the google JS style guide also states it (https://google.github.io/styleguide/jsguide.html#naming-local-variable-names), I switched over to that.